.gitignore

```diff
@@ -2,4 +2,4 @@ node_modules/
 .DS_Store
 dist/
 coverage/
-src/__snapshots__/*
\ No newline at end of file
+*__snapshots__*
```

Comment:
Please consider adding a new line to this file 
<img width="270" alt="Screenshot 2019-07-13 at 5 48 07 PM" src="https://user-images.githubusercontent.com/43774983/61173043-6fbf5b80-a596-11e9-8832-cea75a56a77e.png">
As you can see here

Reply:
@jennizalwango kindly that has been done